rpc-jobs/multi-node-aio.yml

```diff
@@ -15,21 +15,50 @@
           DEFAULT_IMAGE: "16.04.2"
       - trusty:
           DEFAULT_IMAGE: "14.04.5"
+    trigger:
+      - periodic:
+          CRON: "H H(0-8) * * 1-5"
+      - post-merge:
+          CRON: ""
     exclude:
       - series: mitaka
         context: xenial
     jobs:
-      - 'OnMetal_Multi_Node_AIO_{series}-{context}'
+      - 'OnMetal_Multi_Node_AIO_{series}-{context}-{trigger}'
+      - 'OnMetal_Multi_Node_AIO_Merge_Trigger_{series}'
 
 - job-template:
-    name: 'OnMetal_Multi_Node_AIO_{series}-{context}'
+    name: 'OnMetal_Multi_Node_AIO_Merge_Trigger_{series}'
     project-type: workflow
     properties:
       - build-discarder:
           days-to-keep: 7
+      - rpc-openstack-github
     concurrent: true
     triggers:
-      - timed: "H H(0-8) * * 1-5"
+      - github
+    dsl: |
+      node(){{
+        stage('OnMetal Multi-Node AIO') {{
+          git branch: "{branch}", url: "https://github.com/rcbops/rpc-openstack"
```

Comment:
It will only trigger on the specified branch because it tracks the branch used by the last build.